rudder-web/src/main/webapp/javascript/rudder/angular/apiAccount.js

```diff
@@ -113,7 +113,7 @@ $scope.columnDefs = [
       , 'bSortable': false
       , "sWidth": "10%"
     , "fnCreatedCell" : function (nTd, sData, oData, iRow, iCol) {
-      var color = sData ? "#CCFFCC" : "#FF6655";
+      var color = sData ? "#5cb85c" : "#d9534f";
```

Comment:
I changed the colour here too, even though it's not related to compliance, because it was the same green, and it's better to be consistent across the whole UI.